pueblo/ngr/runner.py

```diff
@@ -237,7 +237,6 @@ def install(self) -> None:
         elif self.has_gradle_files:
             if shutil.which("gradle"):
                 run_command("gradle wrapper")
-            run_command("./gradlew install")
         else:
             raise NotImplementedError("Unable to invoke target: install")
```

Comment:
this error is not in sync anymore.

Reply:
> Don't you need to run a gradle task here? like `./gradlew build`?

I will be happy to try using `build` instead, if this is a more canonical target, considered to be appropriate. Thanks!

> This error is not in sync anymore.

Are you referring to the error message `Unable to invoke target: install"`? Well, this attaches more to the "ngr target", which is still "install" in this case, as it reflects the stage before running the "test" target. Currently, there are only two "high-level" stages of commands emitted by ngr, which are `install` and `test`.

Reply:
In this specific case, if you think `gradle build` is more appropriate than `gradle install`, I will be happy to try that. However, I had the impression that `gradle check` would already invoke `gradle build` implicitly, at least on the projects I was working with. If this is the case, there is no reason to run `gradle build` separately, right?

Reply:
To verify my suggestion, I've just deleted the `build` directories from both of the projects listed below, and just invoked `./gradlew check` on them. It works well.

- https://github.com/crate/cratedb-examples/tree/main/by-language/java-jooq
- https://github.com/crate/crate-sample-apps/tree/main/java-spring

Reply:
> Don't you need to run a gradle task here? like `./gradlew build`?

According to the two samples enumerated above, the answer is "no".

I will be happy to learn more about it, specifically about corresponding behavior on other projects, in order to be able to build a reasonable "most common denominator" thing.

Please note it will be impossible to conceive a single thing suitable for all situations, but there is lots of room for more heuristics and options to be respected by future versions of `ngr`. Only because it is currently surfing the "zero configuration options" wave, does not mean it has to stay like this forever.

Reply:
Ok, I saw the code after this, so yes, check is doing everything, no need to run `build` or `install` separately. thx for double checking!